user-management/src/main/scala/com/normation/plugins/usermanagement/UserManagementUtilities.scala

```diff
@@ -0,0 +1,197 @@
+package com.normation.plugins.usermanagement
+
+import java.io.File
+import java.net.URI
+
+import better.files
+import better.files.Dsl.SymbolicOperations
+import bootstrap.liftweb.{PasswordEncoder, UserConfigFileError, UserFile, UserFileProcessing}
+import com.normation.plugins.usermanagement.UserManagementIO.getUserFilePath
+import com.normation.rudder.Role.Custom
+import com.normation.rudder.{AuthorizationType, Rights, Role, RoleToRights}
+import com.normation.rudder.repository.xml.RudderPrettyPrinter
+import net.liftweb.common.{Box, Failure, Full}
+import org.springframework.core.io.{ClassPathResource => CPResource}
+
+import scala.xml.{Elem, Node, NodeSeq}
+import scala.xml.parsing.ConstructingParser
+import scala.xml.transform.{RewriteRule, RuleTransformer}
+
+case class User(username: String, password: String, role: Set[String]) {
+  def toNode: Node = <user name={ username } password={ password } role={ role.mkString(",") } />
+}
+
+object UserManagementIO {
+
+  def replaceXml(currentXml: NodeSeq, newXml: Node, path: URI): Box[files.File] = {
+    val file: files.File = files.File(path)
```

Comment:
you should only use better.files File and not Java.io file, 

Remove java.io from imports

or at least do

import java.io.{File => JFile} 

so file is from better.files!